File: src/app/Models/Job.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace AT_T_31_10.Models
{
    public class Job
    {
        public int Id { get; set; }
        public int RecruiterId { get; set; }
        public int Experience { get; set; }
        [Required]
        public string Title { get; set; }
        public string Position { get; set; }
        public string Description { get; set; }
        public string Skillset { get; set; }
        public bool Active { get; set;}
        public bool Published { get; set;}
    }
}

File: src/app/ModelService/Job.ts
import { Manager } from "./Manager";
import { Skill } from "./Skill";

export class Job {
    constructor(
        title: string, description: string,
        position: string) {
        this.Experience = 0;
        this.Title = title;
        this.Position = position;
        this.Description = description;
        this.Active = true;
        this.Published = true;
        this.Skills = [];
        this.Recruiters = [];
    }
    Id: number;
    Experience: number;
    Title: string;
    Position: string;
    Description: string;
    Active: boolean;
    Published: boolean;
    Skills: Skill[];
    Recruiters: Manager[];
}


